Add mandolin mixer icon

jplynch019 opened the pull request:
Resolves: *(direct link to the issue)*

*(short description of the changes and the motivation to make the changes)*

<!-- Use "x" to fill the checkboxes below like [x] -->

- [ x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [ x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x ] Each commit's message describes its purpose and effects
- [ x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [ x] Each commit compiles and runs on my machine without known undesirable changes of behavior
- *** Note:  Ubuntu Linux 20.04 LTS build only (not windows or mac)

jplynch019 commented:
Had an issue synch'ing VS Code ... modified MixerBoard.cpp here for proper naming of mandolin_xpm in MixerBoard::LoadMusicalInstruments.  I do not have a build environment for Windows or Mac, just Ubuntu Linux 20.04 LTS .  My local build works and the new logo displays.

LWinterberg commented:
Hello! We just [switched master to 4.0.0](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444) which makes this PR incompatible and/or obsolete as we're switching UI framework and reorganizing large parts of the codebase. Audacity 4 won't be using XPMs, instead we hope to have an icon font with many more of these instruments.

jplynch019 commented:
That's ok.  It seems no one felt compelled to review the PR.  I had the
process down so well where I created and added several instrument's icons
for my private builds.  Good luck with the 4.0 project.  A new UI framework
will result in a flood of bug fixing activity. I will clone the 4.0
environment and try to keep up with the dev progress.

On Tue, Oct 15, 2024, 3:56 AM Leo Wattenberg ***@***.***>
wrote:

> Hello! We just switched master to 4.0.0
> <https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444>
> which makes this PR incompatible and/or obsolete as we're switching UI
> framework and reorganizing large parts of the codebase. Audacity 4 won't be
> using XPMs, instead we hope to have an icon font with many more of these
> instruments.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/audacity/audacity/pull/2556#issuecomment-2413158176>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD23YBQKIUQHHUY5CA4IUQLZ3TDEVAVCNFSM6AAAAABP6QRAG6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMJTGE2TQMJXGY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
